test/integration/targets/ansible/runme.sh

```diff
@@ -15,3 +15,111 @@ ansible-config list | grep 'DEFAULT_REMOTE_USER'
 
 # 'view' command must fail when config file is missing
 ansible-config -c ./ansible-non-existent.cfg view && exit 1 || echo 'Failure is expected'
+
+################################################################################
+# BEGIN: Test case for ansible-galaxy
+#
+
+# Prep git, becuase it doesn't work inside a docker container without it
+git config --global user.email "tester@ansible.com"
```

Comment:
Using `--global` is dangerous, as it overwrites existing configuration. Avoid it if possible. If not, only set the values if they're not already set. Also make sure to add the `destructive` alias for the test target.

Reply:
@mattclay it runs in a docker container for the test, what's the potential fall out from `git config --global`?

Reply:
As discussed in IRC, the reason is that not everyone runs the tests in disposable docker containers. Tests that make changes outside the temporary directories of a test run should be as non-destructive as is practical and set the `destructive` alias to indicate that they do make such changes.

Another reason that I forgot to mention earlier is that it's always better if tests don't leave behind global changes when they finish, since those changes could interfere with a following test, even when using disposable containers. New containers are used per test run, not per test.